dotcom-rendering/src/web/layouts/NewsletterSignupLayout.tsx

```diff
@@ -101,6 +115,77 @@ const leftColWrapperStyle = css`
 	}
 `;
 
+const previewButtonWrapperStyle = css`
+	padding: ${space[2]}px 0;
+	${from.desktop} {
+		display: none;
+	}
+`;
+
+const mainGraphicWrapperStyle = css`
+	border-radius: ${space[2]}px;
+	overflow: hidden;
+	margin: ${space[4]}px 0;
+
+	& > figcaption {
```

Comment:
We don't typically use these types of selectors in DCR. Instead, we favour using components and having local styles applied there. One of the goal of DCR is componentised, isolated code. This is also true for styles but by having css selectors that also apply styles to descendants you create a coupling that can cause problems when scaling things.

I think you can simply create a new  css property - captionStyles - and add this there and then target it directly.